### PR TITLE
Add roomba total cleaned area sensor

### PR DESCRIPTION
**Pull request by Xitee1**
## Proposed change

Add roomba total cleaned area sensor.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/106640
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/roomba.markdown [hunk 1]
```diff
@@ -57,6 +57,7 @@ Sensors:
 - **Total missions**: The total number of all missions
 - **Scrubs**: Total number of times the robot has executed a "scrub"
 - **Total cleaning time**: How many hours the robot has spent cleaning in total
+- **Total cleaned area**: The total amount of m² that Roomba has cleaned
```
Comment (davet2001) [inline source/_integrations/roomba.markdown]:
```suggestion
- **Total cleaned area**: The total area in m² the robot has cleaned
```
Naming of the robot should be consistent with previous bullets.

Will the units be subject to localisation e.g. square feet? If so maybe omit m^2 here.

**Review by davet2001 — CHANGES_REQUESTED** [review]
Minor suggestions.

File: source/_integrations/roomba.markdown (hunk 1 above)
Comment (Xitee1) [reply source/_integrations/roomba.markdown]: I don't think localisation of this sensor will be possible in near future. See: https://github.com/home-assistant/architecture/discussions/849

Comment (davet2001) [reply source/_integrations/roomba.markdown]: Ok, makes sense.

**Review by davet2001 — APPROVED** [review]
Looks good to me. Based on the timing of the parent PR merge, I think this will be available in 2024.2

**Comment by c0ffeeca7** [comment]
Are you planning to make any more changes to this PR, @Xitee1? Asking because it is in _Draft_ status.

**Comment by Xitee1** [comment]
No I'm not. I haven't noticed that it got changed to draft.

**Pull request merged**